spring-boot-project/spring-boot-dependencies/pom.xml

```diff
@@ -138,7 +138,7 @@
 		<mssql-jdbc.version>6.4.0.jre8</mssql-jdbc.version>
 		<mysql.version>8.0.13</mysql.version>
 		<nekohtml.version>1.9.22</nekohtml.version>
-		<neo4j-ogm.version>3.1.5</neo4j-ogm.version>
+		<neo4j-ogm.version>3.2.0-alpha02</neo4j-ogm.version>
```

Comment:
We cannot really move to an `alpha` version at this point. Can you clarify how this is scoped with the next Spring Data release train?

Reply:
The final version of 3.2 is due to be released with / for Spring Data Moore. It has several new features (this here being one of it) but more important, we removed stuff we deprecated in 3.1.5 and 3.1.6. 

The reason for us using it in this PR is that we wanted to get at least the proposed changes into a discussion.

Neo4j releases those alphas into central btw, so no need to add additional repos (if this is good or not is another question).

Reply:
Thanks and good to hear the release will be aligned with the `Moore` release train.